sycl/test/esimd/regression/esimd_hpp_wo_sycl_hpp_included.cpp

```diff
@@ -0,0 +1,10 @@
+// RUN: %clangxx -fsycl -fsyntax-only %s
+// expected-no-diagnostics
+
+#include <sycl/ext/intel/experimental/esimd.hpp>
+
+#include <limits>
```

Comment:
nit: those includes are not needed probably.

Reply:
right!